tests/test_wave_03.py

```diff
@@ -2,22 +2,26 @@
 
 from adagrams.game import score_word
 
+# @pytest.mark.skip
```

Comment:
No big deal, but since we didn't make any substantive changes to this file, it didn't need to be committed.